src/selectors/Entities/nameNote.js

```diff
@@ -11,3 +13,51 @@ export const selectNameNoteIsValid = state => {
   const { isValid = false } = nameNoteState ?? {};
   return isValid;
 };
+
+export const selectClosestPCDToVaccination = (patient, vaccinationEntryDate) => {
```

Comment:
A short comment what closest, i.e. closest before vaccination date, means could be helpful.

Reply:
Thanks - agree that it was a little unclear before! d3c70660686444c16475bb41495d8eb7093a1a90